cactusbot/handlers/events.py

```diff
@@ -98,12 +108,33 @@ def __init__(self, cache_data, api):
                     "%USER%", packet.user
                 ))
 
+    async def on_join(self, packet):
+        """Handle join packets."""
+
+        if self.alert_messages["join"]["announce"]:
+            return MessagePacket(
```

Comment:
We should eventually add a `level` config value, so that people could optionally just whisper these to people. That's for Later™, though.